client/profile-wizard/steps/business-details/data/extension-benefits.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+export const extensionBenefits = [
```

Comment:
There was a lot of static data inside the original business details component, which made it hard to read the component (it was close to 1k lines) but also we need some of this data in the new flow as well, so I've moved them into their own folder.